src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilationResult.cs

```diff
@@ -13,119 +11,49 @@ namespace Microsoft.AspNet.Mvc.Razor
     /// </summary>
     public class CompilationResult
     {
-        private Type _type;
-
         /// <summary>
-        /// Creates a new instance of <see cref="CompilationResult"/>.
+        /// Creates a new instance of <see cref="CompilationResult"/> that represents a success in compilation.
```

Comment:
- and end with "a successful compilation"
- similarly, "for a failed compilation" below